NEWS.md

```diff
@@ -0,0 +1,57 @@
+# News / Release Notes
+
+## 3.0.0
+
+*Release Date: 2020-Feb-13*
+
+This is a major new release. Highlights:
+
+* Python 3.6+ compatible only. Releases >=3.0.0 will no longer support Py 2.7. Little or no maintenance will be done
+on versions <3.0.0 (which remains Py 2.7 compatible).
+* Enable use of variable schema name, specified by environment variable `PYCDS_SCHEMA_NAME`.
+* Establish database migration control using Alembic.
+* Add migrations that set up standard PyCDS tables, functions, views, and materialized views.
+* Pin dependency versions.
+
+Major / dominant changes:
+
+* [Add migration version-check function](https://github.com/pacificclimate/pycds/pull/58)
+* [Add migration for weather anomaly materialized views](https://github.com/pacificclimate/pycds/pull/55)
+* [Add migration for utility views](https://github.com/pacificclimate/pycds/pull/53)
+* [Add migration for functions ](https://github.com/pacificclimate/pycds/pull/52)
+* [Initialize migration management](https://github.com/pacificclimate/pycds/pull/50)
+* [Make PyCDS schema name agnostic](https://github.com/pacificclimate/pycds/pull/44)
+* [Rationalize test fixtures](https://github.com/pacificclimate/pycds/pull/42)
+* [Drop support for Python <3.6](https://github.com/pacificclimate/pycds/pull/40)
+* [Modify PyCDS table definitions to bring it to canonical current state of CRMP](https://github.com/pacificclimate/pycds/pull/35)
+
+Minor / superseded changes:
```

Comment:
I'm not really sure what you mean by "superseded" changes here. I would understand superseded things as being *removed*, because they're superseded by some other change. But it seems that you're referring to these as being added? I'm confused.

Reply:
Superseded means that the functionality they provided has been provided or otherwise rendered unnecessary by later added features. This release rolls up a *lot* of changes, some from quite long ago which have been superseded in this sense. Possibly I should remove reference to them, as they aren't really relevant to what is in the release now.